src/e2e-tests/rnrtest.ts

```diff
@@ -7,7 +7,7 @@ import { spawn } from "child_process";
 import getPort from "get-port";
 import { test as base } from "@playwright/test";
 
-export const rnrtest = base.extend<{ rnrServer: null }, { rnrPort: number }>({
+export const RnRTest = base.extend<{ rnrServer: null }, { rnrPort: number }>({
```

Comment:
Similarly, this should also be lowercased. Feel free to rename to something more sensible, like `uiTest`, `browserTest`, or something else.